pogoiv/iv_calculator.py

```diff
@@ -77,13 +77,45 @@ def get_ivs(self, pokemon_name, current_cp, current_health, dust_to_upgrade, pow
 
         return possible_stats
 
-    def get_ivs_across_powerups(self, pokemon_name, powerup_stats):
+    def appraisal_filter(self, response, appraisal):
+        filtered_response = []
+        """ Filters the resulting iv set according to the appraisal values. """
+        """ Global appraisal ranges. """
+        percentajes = [(0.0,48.9), (51.1,64.4), (66.7,80.0), (82.2,100.0)]
```

Comment:
Prefer "percentages" since the coding style for this library is english var names.

Pretty minor, but also spaces preferred after tuple commas e.g. (0.0, 48.9) as opposed to the current (0.0,48.9)